src/aws_encryption_sdk_cli/compatability.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Contains logic for checking ESDK and Python Version"""
+import warnings
+
+
```

Comment:
Why is there no Python version warning for this branch?

Reply:
I should not have added one to the 1.x branch.
The ESDK will emit a Python version warning. 
In fact... we should probably cancel these `warning` PRs... and make them only the ESDK...